### PR TITLE
dark theme

### PR DESCRIPTION
**Pull request by WuTheFWasThat**

### CODE REVIEW

**Comment by WuTheFWasThat** [comment]
whoops i forgot to do the keybindings div

File: assets/css/settings.coffee [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    settings = this
+    @mainDiv.find("#theme-selection").on 'input', (e) ->
+      theme = @value
+      settings.data.setSetting('theme', theme)
+      settings.changeStyle(theme)
+  loadRenderSettings: () ->
+    theme = @getSetting 'theme'
+    @changeStyle(theme)
+    @mainDiv.find("#theme-selection").val theme
+  getSetting: (setting) -> # TODO: make data.coffee populate default settings directly during setup instead
```
Comment (vanceza) [inline assets/css/settings.coffee]: Jeff, thoughts on how to handle making the datastore load process use a 'default settings' object? The current way certainly works but I'd rather just write default settings to the datastore if they're not there yet.

**Comment by vanceza** [comment]
One minor internal issue (comment above) aside, switchable themes are working.

**Comment by vanceza** [comment]
(This is now "themes and settings" I guess)

File: assets/css/utils.sass [hunk 2]
```diff
@@ -28,3 +44,40 @@
 .hidden
   display: none
   visibility: hidden
+
```
Comment (WuTheFWasThat) [inline assets/css/utils.sass]: i know the name may not be great, but i think of utils as generic useful things - I'm gonna move the .modal-dialog stuff to view.sass

Comment (WuTheFWasThat) [inline assets/css/utils.sass]: oh i see, this is a generic modal.  well.. i'm gonna move some of the specific stuff like background color, at least, heh

**Comment by WuTheFWasThat** [comment]
merged!  i don't like the way the modal causes the hash to change but oh well lol (just me being OCD)

**Pull request closed without merging**